tests/e2e/util.go

```diff
@@ -625,6 +630,15 @@ func verifyVolumeMetadataInCNS(vs *vSphere, volumeID string,
 			kubernetesMetadata.EntityName != PersistentVolumeClaimName {
 			return fmt.Errorf("entity PVC with name %s not found for volume %s", PersistentVolumeClaimName, volumeID)
 		}
+		if kubernetesMetadata.EntityType == "POD" && kubernetesMetadata.ClusterID != clusterID {
+			return fmt.Errorf("clusterID %s not found for pod %s", clusterID, PodName)
```

Comment:
can we say cluster ID is not matching here
also I thought `kubernetesMetadata.ClusterID` was an array

Reply:
Cluster Id field is still coming as string as of now